graphs/topological_warshall.py

```diff
@@ -0,0 +1,38 @@
+#Topological ordering
+#In topological sorting, a temporary stack is used with the name “s”. The node number is not printed
+#immediately; first iteratively call topological sorting for all its adjacent vertices, then push adjacent
+#vertex to stack. Finally, print contents of stack. Note that a vertex is pushed to stack only when all of
+#its adjacent vertices (and their adjacent vertices and so on) are already in stack.
+
+
+n=int(input("Enter the number of nodes:"))
```

Comment:
Avoid single letter variable name. If you need to use `input`, add `.strip()` at the end to deal with whitespaces in user input.

Reference [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md)